### ISSUE TITLE
Duplicate report

### ISSUE THREAD

**Issue by rpkilby** [terrencepreilly/darglint#18]
This isn't super critical, but it looks like error reporting is repeated?

Function:
```python
def get_envlist(ini, factors):
    """Get the list of env names from the tox config that match the factors.

    See `match_envs` for more details on env name matching.

    Args:
        ini: The parsed tox ini file. This should be the `._cfg` attribute of
            the `config` passed to the `tox_configure` hook.
        factors: The list of env factors to match against.

    """
    ...
```

flake8+darglint output
```shell
lint run-test: commands[0] | flake8 src
src/tox_factor/factor.py:5:1: I201 Missing "Returns" in Docstring: - return
src/tox_factor/factor.py:5:1: I201 Missing "Returns" in Docstring: - return
ERROR: InvocationError for command /Users/bagel/Documents/projects/tox-factor/.tox/lint/bin/flake8 src (exited with code 1)
```

----

The repo is currently private, but I can link/reproduce in a bit when I make it public (is about to go live).

**Comment by rpkilby** [comment]
Note that this is using the flake8 integration. If I call darglint directly, the error only appears once.

```shell
(.venv) Sherpa-2:tox-factor bagel$ darglint src/tox_factor/factor.py 
src/tox_factor/factor.py:get_envlist:4: I201: - return
```

**Comment by terrencepreilly** [comment]
Ah, you're right.  I'll take a look at it tomorrow and see if I can get a fix in.  Thanks for reporting it!

**Comment by terrencepreilly** [comment]
Okay, I've submitted a fix for this.  It turns out that if you have multiple entry points for flake8, it runs the entry point multiple times.  But, you can't have different error codes without having multiple entry points.  If you only have one entry point, it only displays messages that start with that character sequence.  But, if you have more than one, *both* of the entry points display errors with *either* of the character sequences.  It looks like a bug should be submitted to flake8 -- if I have time, I'll try to do that as well.

Resolved in v0.5.4

**Comment by rpkilby** [comment]
Neat solution. Thanks for the fix.
